requirements.txt

```diff
@@ -1,10 +1,8 @@
-click == 7.1.2
+colorama == 0.4.4
+numpy == 1.17.3
 opencv-python == 4.1.2.30
+pyyaml >= 5.3.1
 requests == 2.24.0
-shapely == 1.7.1
 tensorflow == 2.2.0
 tqdm == 4.45.0
-numpy == 1.17.3
-pyyaml >= 5.3.1
-colorama == 0.4.4
-scipy
```

Comment:
`scipy` is a dependency of `tensorflow==2.2.0` but got removed in the recent versions (not found in `tensorflow2.6.0`). So if we are installing via `requirements.txt`, `scipy` should get installed regardless.

But `scipy` is necessary in `setup.cfg` because a `tensorflow` which does not depend on `scipy` may get installed and we still need `scipy` in `decode_multi.py`.

However, I can add it back to `requirements.txt` if you feel it's more consistent that way.

Reply:
Okay, let's add it back for the time being.

Reply:
Added https://github.com/aimakerspace/PeekingDuck/pull/510/commits/2aadec58958ea43713465f32eb3ae15f32e6e694